src/main/java/moze_intel/projecte/events/FovChangeEvent.java

```diff
@@ -1,23 +0,0 @@
-package moze_intel.projecte.events;
-
-import cpw.mods.fml.common.eventhandler.SubscribeEvent;
-import cpw.mods.fml.relauncher.Side;
-import cpw.mods.fml.relauncher.SideOnly;
-import moze_intel.projecte.gameObjs.ObjHandler;
-import net.minecraft.item.ItemStack;
-import net.minecraftforge.client.event.FOVUpdateEvent;
-
-@SideOnly(Side.CLIENT)
-public class FovChangeEvent 
-{
-	@SubscribeEvent
-	public void onFOVChange(FOVUpdateEvent event)
-	{
-		ItemStack legs = event.entity.getCurrentArmor(1);
-		
-		if (legs != null && legs.getItem() == ObjHandler.gemLegs)
-		{
-			event.newfov -= 0.5f;
```

Comment:
This was removed because it was a stupid, stupid attempt to compensate for the speed boost the the boots gave you.
It never worked and worked in a hacky way (what if other mods modify the fov?).
If you put the pants on without the boots, then your fov would shrink to unplayable narrowness.
I opted to just remove it, as the fov change from the boots is not even that significant